src/config/ch.sbb.direktverbindungen/DvLineTitle/DvLineTitle.js

```diff
@@ -0,0 +1,129 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { Feature } from 'ol';
+import { makeStyles, IconButton } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import ZoomOutMapIcon from '@material-ui/icons/ZoomOutMap';
+import DirektverbindungenLayer from '../../../layers/DirektverbindungenLayer';
+import { ReactComponent as TrainIconDay } from '../../../img/train-day.svg';
+import { ReactComponent as TrainIconNight } from '../../../img/train-night.svg';
+
+const useStyles = makeStyles((theme) => ({
+  titleWrapper: {
+    padding: 0,
+    display: 'flex',
+    alignItems: 'center',
+  },
+  titleIcon: {
+    marginRight: 10,
+  },
+  title: {
+    width: '100%',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  zoomOnLineBtn: {
+    '&:hover': {
+      color: theme.palette.secondary.main,
+    },
+  },
+}));
+
+const propTypes = {
+  feature: PropTypes.instanceOf(Feature).isRequired,
+  layer: PropTypes.instanceOf(DirektverbindungenLayer).isRequired,
+  isNightTrain: PropTypes.bool.isRequired,
+  title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+  active: PropTypes.bool.isRequired,
+  teaser: PropTypes.bool,
+};
+
+const defaultProps = {
+  teaser: false,
+};
+
+const DvLineTitle = ({
+  feature,
+  layer,
+  isNightTrain,
+  title,
+  active,
+  teaser,
+}) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const map = useSelector((state) => state.app.map);
+  const topic = useSelector((state) => state.app.activeTopic);
+  const screenWidth = useSelector((state) => state.app.screenWidth);
+  const isEmbeddedTopic = useMemo(() => {
+    return /(-iframe|\.sts)$/.test(topic.key);
+  }, [topic]);
+  const isMobile = useMemo(() => {
+    return ['xs'].includes(screenWidth);
+  }, [screenWidth]);
+  return (
+    <div className={classes.titleWrapper}>
+      {isNightTrain ? (
+        <TrainIconNight className={classes.titleIcon} />
+      ) : (
+        <TrainIconDay className={classes.titleIcon} />
+      )}
+      <span className={classes.title}>
+        {active ? (
+          <>
+            <b>{title}</b>
+            <IconButton
+              className={classes.zoomOnLineBtn}
+              title={t('Auf Linie zentrieren')}
+              onClick={(evt) => {
+                if (!teaser) {
+                  evt.stopPropagation();
+                  evt.preventDefault();
+                }
+                const cartaroFeature = layer.allFeatures.find(
+                  (feat) => feat.get('name') === feature.get('name'),
+                );
+                const view = map.getView();
+                const geom = cartaroFeature.getGeometry();
+                const extent = view.calculateExtent();
```

Comment:
Is extent used ?

Reply:
Yes, to adjust the centering https://github.com/geops/trafimage-maps/pull/1075/files/7208e9ca245666293050489f2bbf28baa25f02c5#diff-7276f4ad871237574b069fab4c28f94eb4017ef2b0dbefc33fe2be10ab6212acR95

Reply:
yes you modify it but your never use it after .